src/config.ts

```diff
@@ -10,7 +10,8 @@ export interface Config {
   root?: string,
   eveRoot?: string,
   internal?: boolean,
-  mode?: Mode
+  mode?: Mode,
+  initJsonDB? : any
```

Comment:
If we can give a specific type to this we should. Is there a reason it needs to be `any`?